Sources/NIOChatClient/main.swift

```diff
@@ -44,7 +44,7 @@ private final class ChatHandler: ChannelInboundHandler {
 let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
 let bootstrap = ClientBootstrap(group: group)
     // Enable SO_REUSEADDR.
```

Comment:
just realise that all the `// Enable SO_REUSEADDR.` and similar comments aren't quite right anymore :)

Reply:
That's still what happens behind the scenes so they're not inaccurate.